lib/gap_intelligence/models/header.rb

```diff
@@ -0,0 +1,13 @@
+module GapIntelligence
+  class Header < Record
+    attribute :name
+
+    def initialize(attrs = {})
```

Comment:
I see, I would not override `initialize` method so that we don't have extra code that will bring same result as we don't have it, also feel confident that `@raw` should keep original raw values, not just name. Team? @GapIntelligence/dev-team